src/monitor.ts

```diff
@@ -132,13 +134,32 @@ export const monitorOpenAI = (
     };
   };
 
-  openAIApi.createCompletion = patchCompletion(
-    openAIApi.createCompletion.bind(openAIApi),
-  );
-  openAIApi.createChatCompletion = patchChatCompletion(
-    openAIApi.createChatCompletion.bind(openAIApi),
-  );
-  openAIApi.createEmbedding = patchEmbedding(
-    openAIApi.createEmbedding.bind(openAIApi),
-  );
+  const createCompletion = OpenAIApi.prototype.createCompletion
+  const createChatCompletion = OpenAIApi.prototype.createChatCompletion
+  const createEmbedding = OpenAIApi.prototype.createEmbedding
+
+  const createCompletionOverRide = function (args: any) {
```

Comment:
You have a typo, should be `createCompletionOverride`.
Same in the function bellow.